### ISSUE TITLE
Sequential blank lines between lists should separate lists

### ISSUE THREAD

**Issue by mojavelinux** [asciidoctor/asciidoctor#2639]
Sequential lists have a strong affinity, so they often get continued or combined when adjacent, which can be surprising. If two sequential lists have more than one blank line separating them, the first list should be terminated.

```
* first unordered list


* second unordered list
```

This rule should also apply for lists of different types:

```
* unordered list


. ordered list
```

The current workaround is to use blank line followed by a line comment above the second list.

```
* first unordered list

//-
* second unordered list
```

**Comment by elextr** [comment]
Agreed that there is a technical omission in the Asciidoc syntax to explicitly begin/terminate lists.  But in most cases its not an issue for writers.

And since one list followed immediately by another with no introduction or explanation is usually bad documentation practice it shouldn't occur often, and (IMHO) is often best avoided by improving your document with an introduction to the following list.

In general in computer language design practice using whitespace as syntax (especially how _many_) is usually avoided since its too easy for users to overlook.

And since this use-case should be rare and unusual the syntax should be visible.  IMO the two blank line syntax is not obvious enough for rare case use.  

Whilst the line comment workaround is unglamourous it can be obvious, especially when you do:

```
* list 1

// new list
* list 2
```

So I don't think double blank lines is a good choice of syntax or even really needed syntax if the line comment works.

**Comment by mojavelinux** [comment]
> since one list followed immediately by another with no introduction or explanation is usually bad documentation practice it shouldn't occur often

You'd be surprised how often this happens. And it's not always in a finished document. This can be a real problem during drafting when the document structure isn't fully filled out.

**Comment by mojavelinux** [comment]
> So I don't think double blank lines is a good choice of syntax or even really needed syntax if the line comment works.

I checked the Markdown rules and it turns out that sequential blank lines between list items are not significant. So there's some precedence here to not do this (someone can't say "but it works in Markdown"). That's something to take into consideration.

**Comment by elextr** [comment]
Yes certainly it can happen in WIP documents, but then your comment should say `// TODO list introduction here` :grin:

I think if syntax is added it should be to indicate list start not list termination.  There is currently the issue that its not possible to specify attributes for both the whole list and the first element separately.

If there was a list start syntax element then that would allow attributes that are to be applied to all list elements by applying them to the list start syntax and then attributes applied to the first element would be for it alone.  Would need some thought about what was suitable syntax through.
